pcsx2/vtlb.cpp

```diff
@@ -559,12 +567,35 @@ static void vtlbUnmappedVWriteSm(u32 addr, OperandType data) { vtlb_Miss(addr, 1
 static void TAKES_R128 vtlbUnmappedVWriteLg(u32 addr, r128 data) { vtlb_Miss(addr, 1); }
 
 template <typename OperandType>
-static OperandType vtlbUnmappedPReadSm(u32 addr) { vtlb_BusError(addr, 0); return 0; }
-static RETURNS_R128 vtlbUnmappedPReadLg(u32 addr) { vtlb_BusError(addr, 0); return r128_zero(); }
+static OperandType vtlbUnmappedPReadSm(u32 addr) {
+	vtlb_BusError(addr, 0);
+	if(!CHECK_EEREC && CHECK_CACHE && CheckCache(addr)){
+		switch (sizeof(OperandType)) {
```

Comment:
Can you clang format the changes so the formatting is consistent, can also do it on the entire file.